tools/deploy_translatable_strings.sh

```diff
@@ -89,6 +89,8 @@ cp "$SOURCE_REPO_ROOT/qiskit_pkg/setup.py" "${TARGET_REPO_ROOT}"
 cat "$SOURCE_REPO_ROOT/requirements-dev.txt" "$SOURCE_REPO_ROOT/requirements-optional.txt" \
     > "${TARGET_REPO_ROOT}/requirements-dev.txt"
 cp "$SOURCE_REPO_ROOT/constraints.txt" "${TARGET_REPO_ROOT}"
+# Add commit hash to be able to run the build with the commit hash before the actual release
+echo $GITHUB_SHA > "${TARGET_REPO_ROOT}/qiskit-commit-hash"
 
 echo "+ 'git add' files to target commit"
 git add docs/ setup.py requirements-dev.txt constraints.txt
```

Comment:
```suggestion
git add docs/ setup.py requirements-dev.txt constraints.txt qiskit-commit-hash
```

Need to make sure that we've added the file to the commit as well - sorry, I missed that the first time.  Other than this comment, this looks good to merge to me.

Reply:
Yes! I missed that too 😄 Thank you for catching that